index.html

```diff
@@ -212,12 +212,8 @@ <h2 class="About" >About <br> Myself </h2>
  <section class="form">
```

Comment:
Great work so far @Tshepo45 , I have just to mention that horizontal scrolling mentioned by the previous reviewer is note solved , I noticed that you are using absolute positioning  to align your sections and this is not correct as it is used only if you want to remove a block from the natural flow of the dom and make it overlap other components which is not the case here. Please make sure to solve this problem before requesting the next review